lib/batch/subject.csv.rb

```diff
@@ -1,13 +1,16 @@
-csv = CSV.open('/home/deploy/all_gf_subject.csv', 'wb')
-terms =  GalterGenericFilePresenter.terms + [
-  :depositor, :date_uploaded, :date_modified, :read_access_group,
-  :collection_ids, :id, :events
+@csv = CSV.open('/home/deploy/all_gf_subject.csv', 'wb')
```

Comment:
Why was the `@` necessary here?

Reply:
Because it was outside the scope of the method. I forgot to commit the `@` previously(I made the change on the server, and forgot to include it here).